src/lib/utils/index.ts

```diff
@@ -113,3 +114,17 @@ export async function forChange(
 export function bnSum(amounts: string[]): BigNumber {
   return amounts.reduce((a, b) => bnum(a).plus(b), bnum(0));
 }
+
+/**
+ *  creates a sentence readable list for an array for string
+ *  e.g. 'UBT, DSB and BAL' for [UBT, DSB, BAL]
+ */
+export function formatWordListAsSentence(words: string[]) {
+  if (!words.length) return '';
+  if (words.length >= 2) {
+    const commaSeperatedWords = initial(words);
+    return `${commaSeperatedWords.join(', ')} and ${words[words.length - 1]}`;
```

Comment:
The word `and` would need to be translated, I think the quickest way is probably to pass `t` here and translate it.